src/parquet/file/writer-internal.cc

```diff
@@ -93,7 +107,7 @@ int64_t SerializedPageWriter::WriteDataPage(int32_t num_rows, int32_t num_values
   data_page_header.__set_encoding(ToThrift(encoding));
   data_page_header.__set_definition_level_encoding(ToThrift(definition_level_encoding));
   data_page_header.__set_repetition_level_encoding(ToThrift(repetition_level_encoding));
-  // TODO(PARQUET-593) statistics
+  data_page_header.__set_statistics(ToThrift(page_statistics));
```

Comment:
Possible through manually changing `__isset.statistics` to true and operating directly on the member variable. That said, compilers might be smart enough to do copy elision.